configure.ac

```diff
@@ -1,12 +1,33 @@
 AC_PREREQ([2.61])
 AC_INIT([mstpd], [0.0.4], [https://github.com/mstpd/mstpd])
 
+# bridge-stp must be installed in /sbin/, and the associated scripts and
+# executables should also be installed outside of /usr/.  Set the default
+# exec_prefix so they are installed in the correct places by default.
+# AC_PREFIX_DEFAULT(...) can be used to override the default prefix, but there
+# doesn't seem to be any standard way to override the default exec_prefix, so
+# just do it manually.
+test "x$exec_prefix" = xNONE && exec_prefix=''
```

Comment:
hmm ; ok
that's actually a good point [ about `/sbin/bridge-stp` ]
let's have this as part of the initial commit set

i think all changes to `configure.ac` could be part of the initial commit set